REDCapUITweaker.php

```diff
@@ -290,6 +385,48 @@ function redcap_every_page_top( $project_id = null )
 		}
 
 
+
+		// When a new project has just been created, enable the Data Resolution Workflow if the
+		// option to enable it automatically has been enabled.
+
+		if ( substr( PAGE_FULL, strlen( APP_PATH_WEBROOT ), 22 ) == 'ProjectSetup/index.php' &&
```

Comment:
...see earlier remark about literal numbers in code?  Perhaps str_contains would be more efficient than my first suggestion that involved repeating the text one is looking for in the URL?

Reply:
Again, using `strpos` is perhaps the solution here, as it would match in the correct position unlike `str_contains` and regular expressions are best avoided if a much simpler function will do the job.